app/models/user.rb

```diff
@@ -7,4 +7,10 @@ class User < ApplicationRecord
     uniqueness: {case_sensitive: false}
   has_secure_password
   validates :password, presence: true, length: {minimum: 6}
```

Comment:
` 6`
các giá trị này em khai báo vào file settings nhé

nếu chưa có file này thì search `gem config in rails ` nhé